### PR TITLE
#FN-49 Feat: User should sign up

### PR DESCRIPTION
**Pull request by Niyobuzima**
# PR Description

To be able to use the application and access authenticated routes, Users should be able to Signup and or Register using Sign up forms on the signup route.
Users cannot access the application without logging in without providing an email and password combination
After a user successfully registers, when they click the verification link in the email it should redirect to the deployed frontend login page

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] User successfully signs up/registers.
- [x] User passes appropriate signup parameters
- [x] The user should receive verification link on email upon successful registration

## Description of tasks that were expected to be completed and that are not yet completed

- [x] None

## How this may be tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `yarn i` to install all required dependencies
2. run `yarn run dev`
3. step 3
4. step 4...

## Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case it's appropriate and applicable

## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: src/pages/register.tsx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  if (registered) {
+    return <ConfirmationPage email={formData.email} />;
+  }
+
+  return (
+    <div className="flex justify-center items-center h-screen">
+      <div className="bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4">
+      <div className="text-green-900  mb-3">
+        <h3 className='text-center font-bold'>SIGN UP</h3>
+        <p className="text-center">Create Your Account In Seconds</p>
```
Comment (Honore123) [inline src/pages/register.tsx]: Change text color to black

File: src/pages/register.tsx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+  };
+  if (registered) {
+    return <ConfirmationPage email={formData.email} />;
+  }
+
+  return (
+    <div className="flex justify-center items-center h-screen">
+      <div className="bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4">
+      <div className="text-green-900  mb-3">
+        <h3 className='text-center font-bold'>SIGN UP</h3>
```
Comment (Honore123) [inline src/pages/register.tsx]: Make SignUp font size a little bit bigger

**Review by Illustre13 — APPROVED** [review]
Well Done bro!!

**Review by Illustre13 — COMMENTED** [review]
Well Done bro! 
Kindly squash your commits to 1.

**Review by Honore123 — COMMENTED** [review]
add navbar and footer on your page, and change the background color to blue sky!

**Review by Illustre13 — APPROVED** [review]

**Review by EvelyneUmubyeyi — APPROVED** [review]

**Review by Honore123 — APPROVED** [review]

**Review by dushimeemma — CHANGES_REQUESTED** [review]
I am enabled to signup on my end check your password validations @Niyobuzima

**Review by dushimeemma — APPROVED** [review]

**Pull request merged**